hack/run-locally.sh

```diff
@@ -239,7 +254,13 @@ echo "rebuilding the CNO"
 hack/build-go.sh
 
 # Autodetect the state of the cluster to determine which mode to run in
-if [[ -z "$(ls -A ${CLUSTER_DIR} 2> /dev/null | grep -v install-config.yaml | grep -v .openshift_install | grep -v env.sh)" ]]; then
+if [[ ! -z "${KUBECONFIG}" ]]; then
```

Comment:
I think if `CLUSTER_DIR` is set then we should use that by default. I often have a stray `KUBECONFIG` env var pointing to some no-longer-running cluster:

    danw@p50:~> echo $KUBECONFIG 
    /home/danw/Desktop/kubeconfig-1790652
    danw@p50:~> oc get nodes
    Unable to connect to the server: dial tcp: lookup api.ci-op-pdnwmn0m-endurance-a5b11.llc.devcluster.openshift.com on 10.11.5.19:53: no such host